packages/react/src/lib/VoiceProvider.tsx

```diff
@@ -160,6 +162,9 @@ export const VoiceProvider: FC<VoiceProviderProps> = ({
   > = useCallback(
     (message, err) => {
       stopTimer();
+      if (message === `Received unknown message type`) {
+        setIsSocketUnknownMessageError(true);
```

Comment:
we would need to make sure that this is set back to `false` when the condition is no longer true.

it might be better to infer this state similar to how we do for `isMicrophoneError`, `isSocketError`, and `isAudioError` above